tools/update_cpu_features.zig

```diff
@@ -1055,6 +1055,7 @@ const targets = [_]ArchTarget{
         },
         .branch_quota = 2000,
         .extra_features = &.{
```

Comment:
Perhaps we can avoid writing these manually as well and extract them from `spec.zig`?